.expeditor/scripts/verify/builder-graph-functional.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -euo pipefail
+
+echo "--- Generating signing key"
+hab origin key generate "$HAB_ORIGIN"
+
+# TODO: core/graphviz doesn't appear to support png
```

Comment:
Did this commented block need to be left in?

Reply:
Probably not?  I left it as a reminder we had a test we weren't exercising in CI, but I don't have strong feelings either way.  I'll take a look and see if there's an easy fix, and remove otherwise.